packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -25,6 +26,17 @@ The following Role types are provided and supported:
 | `SUPER_ADMIN`                           | Super Administrator                 |                                       |
 | `USER_ADMIN`                            | Group Administrator                 | [Groups](/docs/reference/api/groups/) |
 
+### IAM-based Standard Role Types
+<ApiLifecycle access="ea" />
+We also support the following IAM-based standard roles:
+| Role type                                               | Label                            | Permissions                                   |
+| :------------------------------------------------------ | :------------------------------- | :-------------------------------------------- |
+| `ACCESS_CERTIFICATIONS_ADMIN` <ApiLifecycle access="ea" /> | Access Certifications Admin | `okta.governance.accessCertifications.manage` |
```

Comment:
@rogertang-okta can you confirm if "Access Certifications Admin" needs to be changed to "Access Certifications Administrator" and same for "Access Requests Admin" -> "Access Requests Administrator"?

Reply:
yes confirmed